zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -274,6 +280,16 @@ impl NetworkUpgrade {
     ) -> Duration {
         NetworkUpgrade::current(network, height).averaging_window_timespan()
     }
+
+    /// Always returns true if `network` is the Mainnet.
+    /// If `network` is the Testnet, the `height` should be at least
+    /// TESTNET_MAX_TIME_START_HEIGHT to return true.
+    /// Returns false otherwise.
+    ///
+    /// Part of the consensus rules at https://zips.z.cash/protocol/protocol.pdf#blockheader
+    pub fn is_time_rule_active(network: Network, height: block::Height) -> bool {
```

Comment:
There are multiple time consensus rules. This function is for the block maximum time rule:
```suggestion
    pub fn is_max_block_time_enforced(network: Network, height: block::Height) -> bool {
```